### PR TITLE
Add venue google map in day view + changelog entry

### PR DESCRIPTION
**Pull request by niconerd**
https://central.tri.be/issues/91610

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:55](https://github.com/moderntribe/the-events-calendar/blob/daff8da22780142095e54371e64f51d422a86582/src/views/day/single-event.php#L55)

> via jenkins/codesniffer-phpcs

File: src/views/list/single-event.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
  *
  * Override this template in your own theme by creating a file at [your-theme]/tribe-events/list/single-event.php
  *
- * @version 4.6.3
+ * @version TBD
```
Comment (bordoni) [inline src/views/list/single-event.php]: We need to change version on the other file.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:55](https://github.com/moderntribe/the-events-calendar/blob/626f7655ab47159a5255dc70e95a31ddcb469db6/src/views/day/single-event.php#L55)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:55](https://github.com/moderntribe/the-events-calendar/blob/c7b141575f6140b33ff3647269ba43242f74bf5e/src/views/day/single-event.php#L55)

> via jenkins/codesniffer-phpcs

**Comment by niconerd** [comment]
Nice catch @bordoni ready for another review!

**Pull request merged**